src/network/networkprotocol.h

```diff
@@ -991,6 +991,7 @@ enum CSMRestrictionFlags : u64 {
 	CSM_RF_READ_NODEDEFS = 0x00000008,    // Disable nodedef lookups
 	CSM_RF_LOOKUP_NODES = 0x00000010,     // Limit node lookups
 	CSM_RF_READ_PLAYERINFO = 0x00000020,  // Disable player info lookups
+	CSM_RF_CAMERA_CONTROL = 0x00000040,   // Disable camera control
```

Comment:
the default value for the node restriction settings should be altered so it includes this also

Reply:
Yes, very important, feature must be disallowed by server as default, obviously.

Reply:
Ah, whoops. I'll fix this.